man/systemd-cryptenroll.xml

```diff
@@ -308,37 +323,44 @@
 
               <row>
                 <entry>9</entry>
+                <entry>kernel-initrd</entry>
                 <entry>The Linux kernel measures all initrds it receives into this PCR.</entry>
                 <!-- Strictly speaking only Linux >= 5.17 using the LOAD_FILE2 protocol, see https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=f046fff8bc4c4d8f8a478022e76e40b818f692df -->
               </row>
 
               <row>
                 <entry>10</entry>
+                <entry>ima</entry>
                 <entry>The IMA project measures its runtime state into this PCR.</entry>
               </row>
 
               <row>
                 <entry>11</entry>
+                <entry>kernel-image-phase</entry>
```

Comment:
It's not "phases of the kernel image". That makes little sense.
It is actually used for two distinct purposes: measurement of kernel, measurement of boot milestones.

What about `kernel-boot`? (I don't think we need to be *exact* in a complicated case like this one. Just give a general hint what the PCR is used for.)

Reply:
> It's not "phases of the kernel image". That makes little sense. It is actually used for two distinct purposes: measurement of kernel, measurement of boot milestones.
> 
> What about `kernel-boot`? (I don't think we need to be _exact_ in a complicated case like this one. Just give a general hint what the PCR is used for.)

I'm very happy lots of us are having inputs on this. Thank you so much sir.